pa11y/webapp/write-report.js

```diff
@@ -30,6 +33,7 @@ const urls = [
   '/events/Wqkd1yUAAB8sW4By',
   '/event-series/WlYT_SQAACcAWccj',
   '/concepts/n4fvtc49',
+  '/guides/YL9OAxIAAB8AHsyv',
```

Comment:
I think it needs to be added to the URL checker as well (if it's new?)

>// Note: if you add a URL to this list, make sure to also add it to the list
// of URLs checked by the URL checker.
//
// See .buildkite/urls/expected_200_urls.txt
//

Reply:
good point, thanks

Reply:
So it is! I know Alex started [some work on that](https://github.com/wellcomecollection/wellcomecollection.org/pull/8817) and I think I thought they were already aligned. All good!